exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/TopN/TopNBatchTest.java

```diff
@@ -159,7 +157,6 @@ public void priorityQueueOrderingTest() throws Exception {
   @Test
   public void sortOneKeyAscending() throws Throwable {
     ClusterFixtureBuilder builder = ClusterFixture.builder(dirTestWatcher)
-      .configProperty(ClassBuilder.CODE_DIR_OPTION, dirTestWatcher.getDir().getAbsolutePath())
       .configProperty(CodeCompiler.ENABLE_SAVE_CODE_FOR_DEBUG_TOPN, true);
```

Comment:
I think this line should be also removed since `ENABLE_SAVE_CODE_FOR_DEBUG_TOPN` should be disabled for tests.

Reply:
Thanks for catching. Removed it.